arangod/IResearch/IResearchLinkMeta.cpp

```diff
@@ -391,9 +392,9 @@ bool IResearchLinkMeta::init( // initialize meta
 
           return false;
         }
-
-        // inserting two identical values for name is a poor-man's boost multiplier
-        _analyzers.emplace_back(analyzer, std::move(shortName));
+        // avoid adding same analyzer twice
+        if (uniqueGuard.emplace(analyzer->name()).second)
```

Comment:
just a stylistic comment:
we always do our `if` statements like this
```
if (...) {
  /* code goes here */
}
```
even if there is only a single statement in the true-condition block.